base/uk.ac.stfc.isis.ibex.ui.dashboard/src/uk/ac/stfc/isis/ibex/ui/dashboard/widgets/TitlePanel.java

```diff
@@ -71,14 +75,27 @@ public TitlePanel(Composite parent, int style, TitlePanelModel model, Font font)
 		title.setToolTipText("Experiment title");
 		
 		Label lblUsers = new Label(this, SWT.NONE);
-		lblUsers.setLayoutData(new GridData(SWT.RIGHT, SWT.CENTER, false, false, 1, 1));
+		lblUsers.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 1, 1));
 		lblUsers.setFont(font);
 		lblUsers.setText("Users:");
 		
 		users = new Label(this, SWT.NONE);
 		users.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
 		users.setFont(font);
 		users.setText("Experiment users");
+		users.setToolTipText("Experiment users");
+
+		btnDisplayTitle = new Button(this, SWT.CHECK);
+		btnDisplayTitle.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
```

Comment:
There is a slight issue with the layout: the new button & label are within the same gridlayout as the title & user readback, and because it is much longer it pushes the actual values off to the right:

![image](https://github.com/ISISComputingGroup/ibex_gui/assets/18398579/c72fa885-6d30-4758-96e6-f1e313290664)

This happens because each row/column will resize to accommodate the largest bit of content inside it, so to fix it we need to decouple the size of the first column from the size of the new button. To do this, you can give the GridData on the button (which is the object that defines how this widget fits inside the parent GridLayout) a `horizontalSpan` argument of 2 like so:

```suggestion
		btnDisplayTitle.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 2,1));
```
This says that `btnDisplayTitle` may span across 2 horizontal cells and so can use the whole width of the title panel rather than just the first column. This should produce a layout looking something like this:

![image](https://github.com/ISISComputingGroup/ibex_gui/assets/18398579/0c35ca35-4d69-4085-a041-2e50b93e7b95)